psutil/_psutil_bsd.c

```diff
@@ -812,6 +862,7 @@ psutil_users(PyObject *self, PyObject *args) {
 #else
     struct utmpx *utx;
 
+    setutxent();
```

Comment:
I seem to remember this won't work on FreeBSD but I may be wrong. Have you tested this on FreeBSD?